src/Gateway.php

```diff
@@ -48,82 +69,82 @@ public function setApiKey($value)
 
     /**
      * @param  array $parameters
-     * @return \Omnipay\Mollie\Message\FetchIssuersRequest
+     * @return AbstractRequest|FetchIssuersRequest
```

Comment:
Isn't it redundant to add `AbstractRequest` as the `FetchIssuersRequest` already extends that? (Same for the others below)